pkg/integrations/execution/v4/runner/runner.go

```diff
@@ -248,28 +248,16 @@ func (r *runner) execute(ctx context.Context, matches *databind.Values, pidWCh,
 		o := out
 		go func(txn instrumentation.Transaction) {
 			defer wg.Done()
```

Comment:
Transaction in different goroutines needs to be handled in a controlled way. Removing these segments for now to avoid errors.